pennylane_lightning/_serialize.py

```diff
@@ -54,45 +55,35 @@
 except ImportError:
     pass
 
+pauli_name_map = {
+    "I": "Identity",
+    "X": "PauliX",
+    "Y": "PauliY",
+    "Z": "PauliZ",
+}
```

Comment:
Is there an equivalent map to this defined elsewhere in PL that we can just use here? If not, I think it may be worth adding as this will need to be replicated for LightningGPU and LightningKokkos too.